### DISCUSSION TITLE
Version 1.0 Reactor Integration

### DISCUSSION THREAD

**Discussion by ovu** [arrow-kt/arrow#2538] [General]
Hi all,

I am starting to migrate to the Version 1.0 and it is really a great Version. I do really like the syntax. However, I am using the Reactor Integration (comprehensions for Flux and Mono) and I am still do not find in the Version 1.0. Am I missing something? Or when do I find the Integration for Reactor?

Thanks and congrats for the Version 1.0!

**Comment by nomisRev** [comment]
Hey @ovu,

The computation we had for `Flux` was sadly not great, and we had therefore removed it. At this moment, there is not really a good replacement for it in Kotlin.

To replace the DSLs for Mono you an use [KotlinX Reactor Integration](https://github.com/Kotlin/kotlinx.coroutines/blob/master/reactive/kotlinx-coroutines-reactor/README.md) and combin that with Arrow Fx Coroutines.
That should give you the same APIs as before.

**Comment by kyay10** [comment]
Closing since Reactor integrations are infeasible for the time being
However, you might want to look at [KonTinuity](https://github.com/kyay10/KonTinuity), which provides computation blocks for any monad you want. It thus can allow for binding `Flux`
